procrastinate/app.py

```diff
@@ -67,6 +67,7 @@ def task(
         queue: str = jobs.DEFAULT_QUEUE,
         name: Optional[str] = None,
         retry: retry_module.RetryValue = False,
+        disable_full_path_warning: bool = False,
```

Comment:
For boolean values, I often find it easier to have positive variables than negative ones, in order to avoid double negatives like `if not disable_full_path_warning` below.

A nice effect is that in "enable_full_path_warning", the "enable" word can be omitted, so `full_path_warning: bool=True`.

Then, as far as I understand, it's called `full_path` because in the code below, it uses a method we've named `full_path`. It's called `full_path` because it concatenates the module path and the name, but from an exterior point view, it's just a path.

I suggest `path_warning: bool = True` which I feel is shorter and as expressive. What do you think ? (it's totally ok if you'd rather go a different route)

One thing we'll need, though, is an explanation in the docstring just below this code. I'm thinking something along the lines of:

```
    path_warning :
        When True (by default), if the name of the task does not match
        the location where the task can be imported, procrastinate will
        emit a logger warning at definition time.
```